client/src/LoginPage.js

```diff
@@ -25,11 +24,13 @@ class LoginPage extends Component {
   }
 
   onSubmit(e) {
+    const audio = '/Users/keving17/Documents/Github/TSL/teacher-moments/server/20180213-185953-485_8d44f67e-d783-4597-aec2-d417325cce06.wav';
```

Comment:
This is some good hacking to get a way to test this out! 👍  I'd suggest trying out new features from a clean scratch space (or at least a new component).  That usually makes it easier to unentangle things and get it into a state you can ship once you get the proof of concept working, but it's always hard to find the shortest path to hacking a prototype :)

If it's helpful at all, https://github.com/mit-teaching-systems-lab/transcriber/blob/master/transcribe.js is an example of how I approached that when I played with a similar thing for Google Cloud Speech API a while back.  At the time the quality wasn't good enough to be usable, but you can see it's just a few lines doing the proof-of-concept and so easier to develop and debug, and then once it's working you can figure out how to include it into the actual app.